src/platform/unix/mod.rs

```diff
@@ -873,50 +922,37 @@ fn recv(fd: c_int, blocking_mode: BlockingMode)
     Ok((main_data_buffer, channels, shared_memory_regions))
 }
 
-#[cfg(target_os="android")]
-fn maybe_unlink(_: *const c_char) -> c_int {
-    // Calling `unlink` on a file stored on an sdcard immediately deletes it.
-    // FIXME: use a better temporary directory than the sdcard via the Java APIs
-    // and threading that value into Servo.
-    // https://code.google.com/p/android/issues/detail?id=19017
-    0
+fn create_string() -> CString {
+    let epoc = UNIX_EPOCH.elapsed().unwrap_or(Duration::new(0, 0));
```

Comment:
I think you can just `unwrap()` here -- time elapsed since epoch should always be valid.

(Weird that there seems to be no simpler way to obtain this... And also weird that there seems to be no way to obtain a global monotonic timestamp. Doesn't really matter though in this case, as with a precise timestamp along with the PID and a counter, a collision is well-nigh impossible even if system time does jump back...)

`epoc` is a weird variable name, BTW... Just `timestamp` I'd say?